xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/resources/ApplicationResources.properties

```diff
@@ -65,4 +65,6 @@ job.question.OverwriteQuestion.lastAuthor=Last author: {0}
 job.question.OverwriteQuestion.lastModified=Last modified: {0}
 job.question.OverwriteQuestion.applyToAll=Apply this action to all the following pages.
 job.question.OverwriteQuestion.replace=Replace
-job.question.OverwriteQuestion.skip=Skip
\ No newline at end of file
+job.question.OverwriteQuestion.skip=Skip
+
+job.waiting=This job is waiting to start, please be patient: another job might be running on the same pages.
```

Comment:
Should probably choose a different work since WAITING is a job state indicating that the job has been started but is currently waiting for an input (generally a question).

Reply:
You mean for the translation key right? job.waitForStarting? job.noneState.display?

Reply:
That works for me. I'll use it.